sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClient.java

```diff
@@ -187,6 +188,22 @@ public Mono<HttpResponse> send(HttpRequest request, Context context) {
                 .onRetryExhaustedThrow((ignoredSpec, signal) -> signal.failure()));
     }
 
+    @Override
+    public HttpResponse sendSync(HttpRequest request, Context context) {
```

Comment:
We should also update the name of the client to `NettyHttpClient` as it's no longer just an async client. Also, since this is not a public class, renaming it is not a breaking change.